buku.py

```diff
@@ -1492,6 +1492,37 @@ def get_page_title(resp):
         return htmlparser.parsed_title
 
 
+def create_poolmanager(url):
+    '''Creates a pool manager with proxy support, if needed.
+    
+    Checks to see if there are proxy variables defined and creates
+    a ProxyManager. If not, creates a PoolManager.
+    '''
+
+    # Check if http_proxy or HTTP_PROXY is defined
+    http_proxy = os.environ.get('http_proxy')
+    if not http_proxy:
+        http_proxy = os.environ.get('HTTP_PROXY')
+
+    # Check if https_proxy or HTTPS_PROXY is defined
+    https_proxy = os.environ.get('https_proxy')
```

Comment:
This and only this - https_proxy. Please remove HTTPS_PROXY, http_proxy and HTTP_PROXY. The last 2 should not be supported in any recent sane app in the modern world. And I don't want have an additional check for HTTPS_PROXY.

From our perspective, we support `https_proxy`. Period. Please update the man page accordingly.